app/javascript/article-form/components/PageTitle.jsx

```diff
@@ -2,11 +2,19 @@ import { h } from 'preact';
 import PropTypes from 'prop-types';
 import { OrganizationPicker } from '../../organization/OrganizationPicker';
 
-export const PageTitle = ({ organizations, organizationId, onToggle }) => {
+export const PageTitle = ({
+  organizations,
+  organizationId,
+  onToggle,
+  previewLoading,
+}) => {
   return (
     <div className="crayons-field__label flex items-center flex-1">
-      <span className="hidden s:inline-block mr-2 whitespace-nowrap">
-        Create Post
+      <span
+        aria-live="polite"
+        className="hidden s:inline-block mr-2 whitespace-nowrap"
+      >
+        {previewLoading ? 'Loading preview' : 'Create Post'}
```

Comment:
I think we need to separate out the `aria-live`. It dynamically announces the changing text inside, so what we have at the moment is:

- "Create post"
- "Loading preview"
- "Create post"

when what we're trying to communicate to screen reader users is:

- Loading preview
- Preview loaded

I think the most straightforward way to do this would be to remove the `aria-live` from this `PageTitle` component (keeping the other changes so the visual title is correct), and instead add to `articleForem.jsx`:

```jsx
<span aria-live="polite" className="screen-reader-only">
  { previewLoading ? "Loading preview" : null }
  { previewShowing && !previewLoading ? "Preview loaded" : null }
</span>
```

Maybe there is a more elegant way to set the inner text content, but essentially I think we want:

- The inner text to be empty if we're currently on the "Edit" tab
- The inner text to be "Loading preview" when in loading state
- The inner text to be "Preview loaded" when on the preview tab but not in edit state

(the "screen-reader-only" class will make sure it's not visible to users, but it will still be announced by screen readers)

Reply:
I have applied this change and this is the output:

https://user-images.githubusercontent.com/9396084/177530102-e0b1a300-e9da-4d24-9a33-d5e40d30ae9f.mov